pkg/feeds/npm/npm.go

```diff
@@ -52,9 +59,14 @@ type PackageEvent struct {
 	Title string `xml:"title"`
 }
 
+type cacheEntry struct {
+	ETag     string
```

Comment:
could we add a little documentation? (yes this package is underdocumented)
e.g. what does the cache entry store exactly, where does the ETag come from, what does versions refer to?

Reply:
Done